spec/open_weather/endpoints/stations_spec.rb

```diff
@@ -94,4 +94,95 @@
       end
     end
   end
+
+  describe '#create_measurements' do
+    it 'creates measurements', vcr: { cassette_name: 'stations/create_measurement_success' } do
+      create_params = {
+        "station_id": '5ed21a12cca8ce0001f1aef1',
+        "dt": 1479817340,
+        "temperature": 18.7,
+        "wind_speed": 1.2,
+        "wind_gust": 3.4,
+        "pressure": 1021,
+        "humidity": 87,
+        "rain_1h": 2,
+        "clouds": [
+          {
+            "condition": 'NSC'
+          }
+        ]
+      }
+      data = client.create_measurements([create_params])
+      expect(data).to be_nil
+    end
+
+    context 'when station does not exist' do
+      it 'raises error', vcr: { cassette_name: 'stations/create_measurement_failed_with_invalid_station' } do
+        create_params = {
+          "station_id": -1,
+          "dt": 1479817340,
+          "temperature": 18.7,
+          "wind_speed": 1.2,
+          "wind_gust": 3.4,
+          "pressure": 1021,
+          "humidity": 87,
+          "rain_1h": 2,
+          "clouds": [
+            {
+              "condition": 'NSC'
+            }
+          ]
+        }
+        expect { client.create_measurements([create_params]) }
+          .to raise_error(OpenWeather::Errors::Fault, /expected=string, got=number/)
```

Comment:
This is an odd error, doesn't tell me that the station was invalid? Should this be a specialized error?

Reply:
This seems to be what the API returns, at least with a number:
https://github.com/dblock/open-weather-ruby-client/pull/21/files#diff-9b98e6c2b2b1880dacb9ff685ba9c676R41

Let me try with a string